DBSCE/Overview.md

```diff
@@ -0,0 +1,275 @@
+
+A clean room enviroment is a reliable, malware and exploit free state of a system. Examples can be: 
+- New device from the factory connected to a secure network during registration. 
+- Company issued devices configured by admin for the employees.
+- A malware free device installing browser in a secure network.
+
+One device registration client can manage multiple devices on the same physical device. There also can be multiple device registration clients on the same device. For example, a user can register a device to their organization as an employee and can simultaneously register the same device for a vendor organization. Both these registrations can either be managed by a single device registration client or through separate device registration clients depending on the implementation and separation between these organizations.
+
+The device registration client can be owned and supported by:
+
+- Operating system - the device gets registered when the OS is installed.
+- Browser - the device gets registered when the browser is installed.
+- Device management software (MDM provider) - the device gets registered when the MDM is enrolled.
+- Third-party software vendor - the device gets registered according to the vendor rules.
+
+DBSC(E) aims to support most of these scenarios. DBSC(E) does not define the device registration protocol, but is only concerned that the device registration is executed in a "clean room" and the management of the generated keys to prove device binding.
+
+The device registration is expected to be a once-in-a-lifetime protected operation, and the user is expected to perform this operation with a clean room environment.
+
+![DeviceRegistration](./DeviceRegistration.svg)
+
+### Local Key Helper
+
+DBSC(E) introduces the concept of `Local Key Helper`.
+
+**Local Key Helper** is an integral part of the the **Device Registration Client**, a software interface responsible for the DBSC Key management. *Local key helper* can be public or private and is expected to be either shipped as a part of a given enterprise framework (with the IDP/OS) or can be installed by a provider in compliance with the protocol expanded below. DBSC(E) defines browser interaction with _Local key helpers_ for each platform. Those details are outlined in [KeyGeneration.md](./KeyGeneration.md).
```

Comment:
>>> **Local Key Helper** is an integral part of the the **Device Registration Client**, a software interface responsible for the DBSC Key management.

Unclear whether we're saying device registration client is responsible for the DBSC key management or local key helper

Reply:
Added clarity.